galaxy.yml

```
@@ -1,15 +1,15 @@
 ---
 namespace: community
 name: aws
-version: 7.0.0-dev0
+version: 7.0.0
 readme: README.md
 authors:
   - Ansible (https://github.com/ansible)
 description: A variety of Ansible content to help automate the management of AWS services.
 license_file: COPYING
 tags: [community, aws, cloud, amazon]
 dependencies:
-  amazon.aws: '>=7.0.0-dev0'
+  amazon.aws: '>=8.0.0-dev0'
```

```suggestion
  amazon.aws: '>=7.0.0'
```

This is still the stable-7 branch, we only need amazon.aws 7.x :)

`>= 8.0.0-dev0` will be needed on the `main` branch, but can't be done without https://github.com/ansible-collections/amazon.aws/pull/1837